### PR TITLE
Responsive Design

### PR DESCRIPTION
**Pull request by oliferwolox**
## Summary

Add responsive design to page. 

## Screenshots

css-responsive

## Trello Card

https://trello.com/c/vQFsxEhw/15-dise%C3%B1o-responsive.

### CODE REVIEW

File: maquetado/setup/scss/footer.scss [hunk 1]
```diff
@@ -23,11 +24,16 @@ $footer-height: 180px;
...
 .footer-text {
   font-size: $text-big;
   margin: 4px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .footer-copyright {
+    align-self: inherit;
+    font-size: $text-smaller;
+    margin-right: 0px;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/footer.scss]: join this margins in one line

File: maquetado/setup/scss/layout.scss [hunk 2]
```diff
@@ -34,3 +34,27 @@
...
+  }
+
+  .tech-container {
+    grid-auto-flow: row;
+    grid-gap: 0px;
+    grid-template-columns: $tech-container-width;
+    margin-left: 0px;
+  }
+
+  .tech-container div:first-child {
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/layout.scss]: same than before, put this inside `tech-container`, we only do this with pseudo elements as you see that maintaining the nesting its quite harder than just writing the styles here

File: maquetado/setup/scss/layout.scss [hunk 3]
```diff
@@ -34,3 +34,27 @@
...
+    border-right: inherit;
+    grid-row:1;
+    height: inherit;
+  }
+
+  .tech-container div:last-child {
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/layout.scss]: same

File: maquetado/setup/scss/layout.scss [hunk 4]
```diff
@@ -34,3 +34,27 @@
...
+    border-bottom: hidden;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/layout.scss]: `border: none` shall do this

File: maquetado/setup/scss/layout.scss [hunk 5]
```diff
@@ -34,3 +34,27 @@
...
+    grid-auto-flow: row;
+    grid-gap: 0px;
+    grid-template-columns: $tech-container-width;
+    margin-left: 0px;
+  }
+
+  .tech-container div:first-child {
+    border-right: inherit;
+    grid-row:1;
+    height: inherit;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/layout.scss]: We usually do `100%` here, tho your choice is not bad

File: maquetado/setup/scss/navbar.scss [hunk 6]
```diff
@@ -23,3 +23,18 @@ $navbar-logo-height: 30px;
...
+  .navbar {
+    justify-content: center;
+  }
+
+  .navbar-logo {
+    margin-left: 0px;
+  }
+
+  .navbar-title {
+    margin-right: inherit;
```
Comment (mmamaniwolox) [inline maquetado/setup/scss/navbar.scss]: Im not sure what do you inherit here 🤔

File: maquetado/setup/scss/layout.scss (hunk 2 above)
Comment (Dorian30) [reply maquetado/setup/scss/layout.scss]: Remeber, avoid using html tags as selectors.

**Pull request merged**